spec/requests/controllers/rooms_controller_spec.rb

```diff
@@ -43,5 +54,13 @@
       room = Room.last
       expect(room.activities.take.user).to eq user
     end
+
+    context 'invalid room' do
+      let(:room) { build(:room, name: '') }
+
+      it 'does not create a room without a name' do
```

Comment:
that's not a helpful test, we know it couldn't save an object if validation fails. You want to test for the outcome, ie redirect + flash